docs/core/accounts.md

```diff
@@ -1,177 +1,143 @@
-
-```shell
-solana rent 15000
-Rent per byte-year: 0.00000348 SOL
-Rent per epoch: 0.000288276 SOL
-Rent-exempt minimum: 0.10529088 SOL
-```
-
-Note: Rest assured that, should the storage rent rate need to be increased at
-some point in the future, steps will be taken to ensure that accounts that are
-rent-exempt before the increase will remain rent-exempt afterwards
+On Solana, all data is stored in what are referred to as "accounts”. The way
+data is organized on Solana resembles a
+[key-value store](https://en.wikipedia.org/wiki/Key%E2%80%93value_database),
+where each entry in the database is called an "account".
+
+![Accounts](/assets/docs/core/accounts/accounts.svg)
+
+## Account
+
+Each account is identifiable by its unique address, represented as 32 bytes in
+the format of an [Ed25519](https://ed25519.cr.yp.to/) `PublicKey`. You can think
+of the address as the unique identifier for the account.
+
+![Account Address](/assets/docs/core/accounts/account-address.svg)
+
+This relationship between the account and its address can be thought of as a
+key-value pair, where the address serves as the key to locate the corresponding
+on-chain data of the account.
+
+## AccountInfo
+
+Every account on Solana has the following structure known as the
+[AccountInfo](https://github.com/solana-labs/solana/blob/master/sdk/program/src/account_info.rs#L19).
+
+![AccountInfo](/assets/docs/core/accounts/accountinfo.svg)
+
+The `AccountInfo` for each account includes the following fields:
+
+- **data**: A byte array that stores the state of an account. If the account is
+  a program (smart contract), this stores executable program code. This field is
+  often referred to as the "account data".
+- **executable**: A boolean flag that indicates if the account is a program.
+- **lamports**: A numeric representation of the account's balance in lamports,
+  the smallest unit of SOL (1 SOL = 1 billion Lamports).
+- **owner**: Specifies the public key (program ID) of the program (smart
+  contract) that owns the account.
+
+As a key part of the Solana Account Model, every account on Solana has a
+designated "owner", specifically a program. Only the program designated as the
+owner of an account can modify the data stored on the account or deduct the
+lamport balance.
+
+<Callout>
+  To store data on-chain, a certain amount of SOL must be transferred to an
+  account. The amount transferred is proportional to the size of the data stored on the account. This concept is commonly referred to as “rent”. However, you can
+  think of "rent" more like a "deposit" because the SOL allocated to an account
+  can be fully recovered when the account is closed.
+</Callout>
+
+## System Program
+
+By default, all new accounts are owned by the
+[System Program](https://github.com/solana-labs/solana/tree/master/programs/system/src).
+The System Program is one of Solana's "Native Programs", which comes "built-in"
```

Comment:
You introduce "Native Programs" without defining it. Recommend defining the different programs you can have before going into the System Program. Could be a small section right before this.

Reply:
These are also known as "Native accounts", so that may be a nice way to start a definition and segue into the System Program.

Reply:
updated with intro and link to https://docs.solanalabs.com/runtime/programs